views/alert.erb

```diff
@@ -1,7 +1,12 @@
-<li class="bad">
-	<h4><%= alert.message['service'] %></h4>
-	<p>
-		<%= alert.message['description']['subject'] %>
-		<%= alert.message['description']['description'] %>
+<li class="bad<% if !alert.message['acknowledged'] %> unacknowledged<% end %>">
+  <h4>
+    <%= alert.message['service'] %>
+    <% if !alert.message['acknowledged'] %>
+      <span class="acknowledged">NEW</span>
```

Comment:
Oh and this class is named `acknowledged` but the condition is for unacknowledged.